### PR TITLE
AdminLogQueryTest: Fix numeric overflow in expression #7722

### LINKED ISSUE #7722: AdminLogQueryTest: Fix numeric overflow in expression
`22 * 365 * 24 * 60 * 60 * 1000` causes an overflow as int-32 multiplication is used (See https://stackoverflow.com/questions/1494862/multiplying-long-values). This can be fixed by using [Calendar](https://docs.oracle.com/javase/7/docs/api/java/util/Calendar.html#add(int,%20int)) method.

### PR DESCRIPTION
**Pull request by josephinex**
AdminLogQueryTest: Fix numeric overflow in expression #7722 #7758
Fixes #7722 

I tried to fix the numeric overflow with the help of Math.multiplyExact and Math.addExact. Please, review and let me know whether it is ok like that. If not, I will use calendar methods or something else.

### CODE REVIEW

**Comment by sujeet14108** [comment]
@josephinex There are trailing spaces on `Line 24` and `Line 26`. Remove that to fix travis build

File: src/test/java/teammates/test/cases/util/AdminLogQueryTest.java [hunk 1]
```diff
@@ -22,7 +22,8 @@ public void testAdminLogQuery() {
         Calendar cal = new GregorianCalendar();
         cal.set(1994, Calendar.MAY, 7, 15, 30, 12);
         long startTime = cal.getTimeInMillis();
-        long endTime = startTime + 22 * 365 * 24 * 60 * 60 * 1000; // about 22 years later
+        cal.add(Calendar.YEAR, 22); // about 22 years later
```
Comment (whipermr5) [inline src/test/java/teammates/test/cases/util/AdminLogQueryTest.java]: The comment can be deleted as the code is now self-documenting :)

File: src/test/java/teammates/test/cases/util/AdminLogQueryTest.java [hunk 2]
```diff
@@ -45,8 +46,9 @@ public void testSetQueryWindowBackward() {
         versionList.add("5-44");
         Calendar cal = new GregorianCalendar();
         cal.set(2016, 4, 7, 15, 30, 12);
-        Long startTime = cal.getTimeInMillis();
-        Long endTime = startTime + 3 * 24 * 60 * 60 * 1000; // 3 days later
+        long startTime = cal.getTimeInMillis();
+        cal.add(Calendar.DAY_OF_MONTH, 3);
```
Comment (whipermr5) [inline src/test/java/teammates/test/cases/util/AdminLogQueryTest.java]: Use `Calendar.DATE` as it's less confusing.

**Comment by josephinex** [comment]
Thanks for taking the time to review the changes!

**Pull request merged**